src/main/webapp/js/instructor.js

```diff
@@ -154,6 +154,22 @@ function setupFsCopyModal() {
     });
 }
 
+// For ajax failure handling
+
+/**
+ * Given an element, replaces the HTML content of the element with an error message prompting 
+ * the user to retry.
+ */
+function displayAjaxRetryMessage($element) {
```

Comment:
The function name can be more specific, since this particular error message is only used in panels/feedback results. It probably can't be used for ajax calls elsewhere.